packages/design/src/animation.mdx

```diff
@@ -0,0 +1,46 @@
+---
+name: Animation
+menu: Design
+route: /animation
+---
+
+# Animation
+## Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+###Reveal
```

Comment:
Need space between the ### and the heading 
![image](https://user-images.githubusercontent.com/39704901/93507525-0eea8c00-f8db-11ea-952c-1e9b61268a53.png)